examples/use_existing_bucket/main.tf

```
@@ -5,5 +5,5 @@ module "aws_eks_audit_log" {
   cloudwatch_regions  = ["us-west-2"]
   cluster_names       = ["example_cluster"]
   use_existing_bucket = true
-  bucket_arn          = "arn:aws:s3:::dev5-cloud-account-registrar-lacework"
-}
\ No newline at end of file
+  bucket_arn          = "arn:aws:s3:::lacework-eks-bucket-8805c0bf"
```

We can probably revert this example change.
```suggestion
  bucket_arn          = "arn:aws:s3:::dev5-cloud-account-registrar-lacework"
```